lib/app.dart

```diff
@@ -20,7 +20,7 @@ class App extends StatelessWidget {
         JobsDashboardScreen.route: (context) => JobsDashboardScreen()
       },
       initialRoute: LoginScreen.route,
-      theme: ThemeConfig.of(context)!.buildThemeData(),
```

Comment:
Why was this nullable in the first place? :))

Reply:
mindless copy pasting from another project 👀